templates/base.html

```diff
@@ -2,20 +2,76 @@
 
 <!DOCTYPE html>
 <html lang="en-gb">
-<meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-<head>
+  <head>
     <title>{% block title %} {% endblock %}| Share Spots</title>
     <!-- Latest compiled and minified CSS -->
-</head>
-
-<body>
+  </head>
 
+  <body>
     {% block layout %} {% endblock %}
 
     <script src="{% static 'main.js' %}" type="module"></script>
-    <link href='https://fonts.googleapis.com/css?family=Monoton|Roboto' rel='stylesheet' type='text/css'>
-    <link href='https://fonts.googleapis.com/css?family=Raleway:300,700,500' rel='stylesheet' type='text/css'>
-</body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Monoton|Roboto"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Raleway:300,700,500"
+      rel="stylesheet"
+      type="text/css"
+    />
+
+    {% if venue_detail.latitude %}
+
+    <!-- Map styles and JavaScript  -->
+
+    <style>
+      .map {
+        position: relative;
+        width: 100%;
+        height: 481px;
+        margin-bottom: 2em;
+      }
+
+      @media (min-width: 1440px) {
+        .map {
+          height: 724px;
+        }
+      }
+    </style>
+
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
+      integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="
+      crossorigin=""
```

Comment:
Is the `crossorigin` attribute necessary if the value is empty?

Reply:
it is copy-pasted link from the Leafletjs website... I see that our <link> tags are in the <body> instead of the <head>?